docs/README.md

```diff
@@ -61,38 +52,49 @@ There are several optional parameters that could be passed into `CreateVolumeReq
 | "encrypted"       |                  |          | Whether the volume should be encrypted or not. Valid values are "true" or "false" | 
 | "kmsKeyId"        |                  |          | The full ARN of the key to use when encrypting the volume. When not specified, the default KMS key is used |
 
-### Topology
-`topology.ebs.csi.aws.com/zone` is the only topology key that represents the availability zone of which a volume is accessible.
+## Prerequisites
+### Kubernetes
+* Kubernetes 1.12+ is required. Although this driver should work with any other container orchestration system that implements the CSI specification, so far it has only been tested in Kubernetes.
 
-To enable topology support on kuberetes, make sure `CSINodeInfo` and `CSIDriverRegistry` feature flags are enabled on both kubelet and kube-apiserver and `CSINodeInfo` CRD is installed on the cluster following [Enabling CSINodeInfo](https://kubernetes-csi.github.io/docs/Setup.html#enabling-csinodeinfo).
+* Enable the flag `--allow-privileged=true` in the manifest entries of kubelet and kube-apiserver.
```

Comment:
For the manifest, ru referring to kops cluster manifest? I think we don't need to mention manifest since not every one is using kops. Probably we can say something like:
```
Enable the flag `--allow-privileged=true` for kubelet and kube-apiserver.
```

Reply:
Manifest is not a word specific to kops. Manifests is a generic term valid for all Kubernetes resources / objects whether the cluster is set by kops or some other tool.

Reply:
This flag is now deprecated no?

Reply:
correct, I just ran into that as well